molecular/src/main/kotlin/com/hartwig/actin/molecular/MolecularInterpreterConfig.kt

```diff
@@ -19,6 +19,7 @@ data class MolecularInterpreterConfig(
     val ensemblCachePath: String,
     val driverGenePanelPath: String,
     val knownFusionsPath: String,
+    val panelSpecificationsFilePath: String?,
```

Comment:
In this optional to ease the rollout of this change?

I imagine in practice we'd always provide this file?

Reply:
It was more for the current EMC case as I guess our first version of the spec file won't be correct for them and they currently don't do panels.